slack-moderate-words/README.md

```diff
@@ -0,0 +1,55 @@
+# slack-moderator-words
+
+slack-moderator-words provides a moderation when positng some specific words, and will let the user know how to write better messages.
+
+## Configuration
+
+slack-moderator-words requires a configuration file, by default called `config.json` in the working
+directory. It must look like this:
+
+```json
+{
+  "signingSecret": "some_slack_signing_secret",
+  "accessToken": "xoxp-some-slack-access-token-these-are-very-long-and-start-with-xoxp",
+}
+```
+
+`signingSecret`, `accessToken` are all values provided by Slack when creating and
+installing the app. Check out the [slack app creation guide][app-creation] for more details.
+
+Also, requires a filter file, by default called `filters.yaml` in the working
+directory. It must look like this:
+
+```yaml
+- triggers:
+  - guys
+  action: chat.postEphemeral
+  message: "May I suggest \"all\" instead when addessing a group of people? Thank you. :slightly_smiling_face:"
```

Comment:
'all' can sound very strange in some sentences depending on how it is used. Its not a 1:1 replacement as sometimes you need to address a subset of people that isn't everyone which all implies. Another recommendation, though an older word, might be 'folks'? (https://www.merriam-webster.com/dictionary/folk, definition 1). I think its a pretty neutral term?

Reply:
I don't think this PR is the right place to discuss this kind of thing. Maybe open a new issue or a thread in Slack.